data/json/snippets/addiction.json

```diff
@@ -0,0 +1,138 @@
+    "text": [ "You're shaking… you need some diazepam!" ]
+  },
+  {
+    "type": "snippet",
+    "category": "addict_diazepam_mild_asleep",
+    "text": [ "You dream that you reach into your pocket and pull out a bottle full of that wonderful diazepam candy…" ]
+  },
+  {
+    "type": "snippet",
+    "category": "addict_diazepam_strong_asleep",
+    "text": [ "You dream about lying in an ocean of benzo boxes.  To your dismay, every single box is empty…" ]
+  },
+  {
+    "type": "snippet",
+    "category": "addict_coke_awake",
+    "text": [ "You feel like you need a bump." ]
+  },
+  {
+    "type": "snippet",
+    "category": "addict_crack_awake",
+    "text": [ "You're shivering, you need some crack." ]
+  },
+  {
+    "type": "snippet",
+    "category": "addict_coke_asleep",
+    "text": [ "You feel like you need a bump." ]
+  },
+  {
+    "type": "snippet",
+    "category": "addict_crack_asleep",
+    "text": [ "You're shivering, you need some crack." ]
+  },
+  {
+    "type": "snippet",
+    "category": "addict_nicotine_mild_awake",
+    "text": [ "You could use some nicotine." ]
+  },
+  {
+    "type": "snippet",
+    "category": "addict_nicotine_strong_awake",
+    "text": [ "You need some nicotine." ]
+  },
+  {
+    "type": "snippet",
+    "category": "addict_nicotine_mild_asleep",
+    "text": [ "You could use some nicotine." ]
+  },
+  {
+    "type": "snippet",
+    "category": "addict_nicotine_strong_asleep",
+    "text": [ "You need some nicotine." ]
+  },
+  {
+    "type": "snippet",
+    "category": "addict_opiate_mild_awake",
+    "text": [ "Your hands start shaking… you need some painkillers." ]
+  },
+  {
+    "type": "snippet",
+    "category": "addict_opiate_mild_asleep",
+    "text": [ "Your hands start shaking… you need some painkillers." ]
+  },
+  {
+    "type": "snippet",
+    "category": "addict_opiate_strong_awake",
+    "text": [ "You feel anxious.  You need your painkillers!" ]
+  },
+  {
+    "type": "snippet",
+    "category": "addict_opiate_strong_asleep",
+    "text": [ "You feel anxious.  You need your painkillers!" ]
+  },
+  {
+    "type": "snippet",
+    "category": "addict_amphetamine_mild_awake",
+    "text": [ "You feel depressed.  Speed would help." ]
+  },
+  {
+    "type": "snippet",
+    "category": "addict_amphetamine_mild_asleep",
+    "text": [ "You feel depressed.  Speed would help." ]
+  },
+  {
+    "type": "snippet",
+    "category": "addict_amphetamine_strong_awake",
+    "text": [ "Your hands start shaking… you need a pick-me-up." ]
+  },
+  {
+    "type": "snippet",
+    "category": "addict_amphetamine_strong_asleep",
+    "text": [ "Your hands start shaking… you need a pick-me-up." ]
+  },
+  {
+    "type": "snippet",
+    "category": "addict_amphetamine_paralysis_awake",
+    "text": [ "You stop suddenly, feeling bewildered." ]
+  },
+  {
+    "type": "snippet",
+    "category": "addict_amphetamine_paralysis_asleep",
+    "text": [ "You stop suddenly, feeling bewildered." ]
```

Comment:
What's this?  Does it wake you if you are sleeping?  Is it like medication induced sleep paralysis (during which you aren't asleep anymore but I guess it's called sleep paralysis because you wake up and can't move)

Reply:
It's the same message as before, and the only effect you get is -300 movement points. Not sure if that's accurate at all, but it looks to me like a "temporary paralysis" effect. It doesn't affect sleep currently, so I guess the sleep snippet here would be just for flavor (unless the character should wake up instead?)

Reply:
I'd say it should wake the character and leave them paralyzed for a minute.   That would be my only change.

Reply:
Cheers, updated the effect. I'm not creative enough to come up with an original message for it, though